azure-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+pool:
+  name: VSEngSS-MicroBuild2019-1ES
+  demands: msbuild
+
+variables:
+  BuildConfiguration: 'Release'
+  BuildPlatform: ''
+
+steps:
+- task: MicroBuildSigningPlugin@2
+  inputs:
+    signType: '$(SignType)'
+    zipSources: true
+    feedSource: 'https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+
+- task: MSBuild@1
+  displayName: '/t:Pack'
+  inputs:
+    solution: MSBuildLocator.sln
+    platform: '$(BuildPlatform)'
+    configuration: '$(BuildConfiguration)'
+    msbuildArguments: '/t:Pack /restore -binaryLogger:$(Build.SourcesDirectory)\Pack.binlog'
+
+- task: CopyFiles@2
+  displayName: 'Collect Symbols'
+  inputs:
+    SourceFolder: src/MSBuildLocator
+    Contents: '**\*.pdb'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)/symbols'
+    CleanTargetFolder: true
+
+- task: PublishSymbols@2
+  displayName: 'Enable Source Server'
+  inputs:
+    SymbolsFolder: '$(Build.ArtifactStagingDirectory)\symbols'
+    SearchPattern: '**/*.pdb'
+    SymbolServerType: TeamServices
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Upload Symbols Artifact'
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)/symbols'
+    ArtifactName: Symbols
+
+- task: CopyFiles@2
+  displayName: 'Copy Bin to drop'
+  inputs:
+    SourceFolder: 'src\MSBuildLocator\bin\Release'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)\bin'
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Upload Bin to drop'
+  inputs:
+    PathToPublish: '$(Build.ArtifactStagingDirectory)\bin'
+    ArtifactName: bin
+
+- task: CopyFiles@2
+  displayName: 'Copy NuGet to drop'
+  inputs:
+    SourceFolder: 'src\MSBuildLocator\bin\Release'
+    Contents: '*.nupkg'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)\pkg'
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Upload NuGet to drop'
+  inputs:
+    PathToPublish: '$(Build.ArtifactStagingDirectory)\pkg'
+    ArtifactName: pkg
+
+- task: PublishPipelineArtifact@1
+  displayName: 'Upload MicroBuildOutputs to drop'
+  inputs:
+    TargetPath: '$(Build.StagingDirectory)\bin'
+    ArtifactName: MicroBuildOutputs
+  continueOnError: true
```

Comment:
The `MicroBuildOutputs` of your test build don't look like the `MicroBuildOutputs` of the existing builds, which I think is masking that something is wrong with the "Enable Source Server" step that should be zipping up the sources.